From 29d24b180dcf33844493c484675df5f57a783a38 Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@suse.de>
Date: Tue, 18 Nov 2008 17:20:22 -0800
Subject: [PATCH 15/15] Staging: comedi: remove PCI wrappers

The PCI wrapper macros are no longer needed, as they are
all for compatiblity with older kernel versions.

Cc: Anders Blomdell <anders.blomdell@control.lth.se>
Cc: David Schleef <ds@schleef.org>
Cc: Frank Mori Hess <fmhess@users.sourceforge.net>
Cc: Ian Abbott <abbotti@mev.co.uk>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/comedi/drivers/mite.h |    2 +-
 drivers/staging/comedi/pci.h          |   31 -------------------------------
 2 files changed, 1 insertion(+), 32 deletions(-)
 delete mode 100644 drivers/staging/comedi/pci.h

--- a/drivers/staging/comedi/drivers/mite.h
+++ b/drivers/staging/comedi/drivers/mite.h
@@ -24,8 +24,8 @@
 #ifndef _MITE_H_
 #define _MITE_H_
 
+#include <linux/pci.h>
 #include "../comedidev.h"
-#include "../pci.h"
 
 #define PCI_VENDOR_ID_NATINST		0x1093
 
--- a/drivers/staging/comedi/pci.h
+++ /dev/null
@@ -1,31 +0,0 @@
-/*
- * linux/pci.h compatibility header
- */
-
-#ifndef _COMPAT_PCI_H
-#define _COMPAT_PCI_H
-
-#include <linux/version.h>
-
-#include <linux/pci.h>
-
-#if LINUX_VERSION_CODE < KERNEL_VERSION(2,6,0)
-#define pci_get_device pci_find_device
-#define pci_get_subsys pci_find_subsys
-#define pci_dev_get(x)	(x)
-#define pci_dev_put(x)
-#endif
-
-#if LINUX_VERSION_CODE < KERNEL_VERSION(2,4,22)
-static inline char *pci_name(struct pci_dev *pdev)
-{
-	return pdev->slot_name;
-}
-#endif
-
-#ifndef DEFINE_PCI_DEVICE_TABLE
-#define DEFINE_PCI_DEVICE_TABLE(_table) \
-	struct pci_device_id _table[]
-#endif
-
-#endif /* _COMPAT_PCI_H */
